CONTRIBUTING.md

```diff
@@ -0,0 +1,192 @@
+repository. Click on
+the <a href="https://github.com/WomenCodingCommunity/WomenCodingCommunity.github.io/fork"><img src="https://i.imgur.com/G4z1kEe.png" height="21" width="21"></a>
+Fork symbol at the top right corner.
+
+**2.** Clone your new fork of the repository in the terminal/CLI on your computer with the following
+command:
+
+```bash
+git clone https://github.com/<your-github-username>/wcc-backend.git
+```
+
+**3.** Navigate to the newly created LinkFree project directory:
+
+```bash
+cd wcc-backend
+```
+
+**4.** Create a new branch:
+
+```bash
+git checkout -b YourBranchName
+```
+
+**5.** Sync your fork or your local repository with the origin repository:
+
+- In your forked repository, click on "Fetch upstream"
+- Click "Fetch and merge"
+
+### Alternatively, Git CLI way to Sync forked repository with origin repository:
+
+```bash
+git remote add upstream git@github.com:Women-Coding-Community/wcc-backend.git
+```
+
+```bash
+git fetch upstream
+```
+
+```bash
+git merge upstream/main
+```
+
+### Additional way to Sync forked repository with origin repository using Git CLI:
+
+```bash
+git remote add upstream git@github.com:Women-Coding-Community/wcc-backend.git
+```
+
+```bash
+git checkout origin main
+```
+
+```bash
+git pull upstream main
+```
+
+```bash
+git push origin main
+```
+
+```bash
+git checkout -b create_my_new_branch_from_main
+```
+
+#### Check out the [Github Docs](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork) to learn more about syncing a forked repository.
+
+**6.** Make your changes to the source code.
+
+**7.** Stage your changes and commit:
+
+```bash
+git add <file/folder>
+```
+
+```bash
+git commit
+```
+
+**8.** Push your local commits to the remote repository:
+
+```bash
+git push origin YourBranchName
+```
+
+**9.** Create
+a [Pull Request](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request)!
+
+**10.** **Congratulations!** You've made your first contribution to [**Women Coding Community
+**](https://github.com/WomenCodingCommunity/wcc-backend)! 🙌🏼
+
+**_:trophy: After this, the maintainers will review the PR and will merge it if it helps move the
+project forward. Otherwise, it will be given constructive feedback and suggestions for the changes
+needed to add the PR to the codebase._**
+
+## Find something to work on
+
+The first step to getting starting contributing is to find something
+to work on. Help is always welcome, and no contribution is too small!
+
+Here are some things you can do today to get started contributing:
+
+1. You can browse the issues labeled
+   as [help wanted](https://github.com/WomenCodingCommunity/WomenCodingCommunity.github.io/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22)
+   and [good first issue](https://github.com/WomenCodingCommunity/WomenCodingCommunity.github.io/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22)
```

Comment:
Here also

Reply:
Since we do not use the above labels for now, I have added just one link - to issues (we can update the document once we label the issues with the labels)